CHANGES.md

```diff
@@ -1,5 +1,9 @@
 # Changes
 
+## master
+
+- Reduce the amount of API calls
```

Comment:
The changelog should also mention debug mode and support for specifying a custom polling interval.

Ideally, adding those would have been a separate pull request (or at least a separate commit).